packages/integration-sdk-runtime/src/execution/dependencyGraph.ts

```diff
@@ -284,6 +297,20 @@ export function executeStepDependencyGraph<
         status = StepResultStatus.FAILURE;
       }
 
+      await context.jobState.flush();
```

Comment:
Interesting. Looks like we now flush after each new step. 😈  advocate: do we want to change the implementation of the maps in `localGraphDataStore`, which tracks `stepId` and `_key` for entities?

Reply:
This is confusing...I had slightly changed the behavior in the previous PR and ended up changing it back to the original in this PR. Should have just rolled back the change in the first PR. See here: https://github.com/JupiterOne/sdk/pull/395/files#diff-8912f1fec8d408545c592a8420e4d837119d01694950f32598718ba7abc57d7aL287